app/web/test/setupTests.ts

```diff
@@ -14,7 +14,17 @@ import i18n from "test/i18n";
 import user from "./fixtures/defaultUser.json";
 
 jest.mock("service");
-jest.mock("next/router", () => require("next-router-mock"));
+jest.mock("next/router", () => {
+  const routerMock = jest.requireActual("next-router-mock");
+  return {
+    ...routerMock,
+    events: {
+      on: jest.fn(),
+      off: jest.fn(),
+      emit: jest.fn(),
+    },
+  };
+});
```

Comment:
I debated just `eslint ignoring` this one. Eslint was mad about `require`, but changing to `import` (I think) does not await the Promise returned by the dynamic import, leading to an undefined value for the mock. This causes downstream code, such as Sentry, to attempt to access properties (like `events`) on the unresolved (or partially resolved) mock, resulting in the TypeError.

So this is my workaround. I could go either way though and just slap an ignore there, but all the tests are passing.